### ISSUE TITLE
PR closed when it shouldn't have been?

### ISSUE THREAD

**Issue by greysteil** [dependabot/dependabot-core#439]
_From @RohanNagar on April 2, 2018 17:9_

Hello,

I was going through Dependabot's PRs this morning and something weird happened. The first PR I merged was RohanNagar/thunder#114, which updated a bunch of dependencies that share the same version number.

Once I merged that, I looked at RohanNagar/thunder#116 and found that Dependabot closed this without rebasing, saying "Looks like these dependencies are updatable in another way, so this is no longer needed.". However, the dependencies in that PR were not updated by the one that was merged or by any other commit.

Is this a bug? (Language is Java)

_Copied from original issue: dependabot/feedback#116_

**Comment by greysteil** [comment]
Hmmm, this does look like a bug. I'll look into it - thanks for reporting.

**Comment by greysteil** [comment]
_From @RohanNagar on April 9, 2018 16:54_

Bug was reproduced this morning: I just merged RohanNagar/thunder#125, and dependabot closed RohanNagar/thunder#124 with the same message, though those dependencies have not been updated on the master branch yet.

Maybe of note: I merged the first PR using the "squash and merge" option, which I don't normally do for Dependabot pull requests. I did this for both this morning and last week because that option was still selected from some other PRs that I merged earlier.

**Comment by sklakegg** [comment]
@greysteil is bug back?

With the following config, every time the schedule runs, it will close the PR with an error every odd time it runs:
_Looks like these dependencies are no longer updatable, so this is no longer needed._

1st run: Creates two PRs 
2nd run: Closes PRs with error
3rd run: Creates two PRs

```
version: 2
registries:
  maven-company:
    type: "maven-repository"
    url: "https://maven.pkg.github.com/company"
    username: "sercret-user"
    password: ${{ secrets.very-secret }}
updates:
  - package-ecosystem: "maven"
    directory: "/"
    registries:
      - "maven-company"
    schedule:
      interval: "weekly"
      day: "wednesday"
      time: "09:00"
      timezone: "Europe/Oslo"
    commit-message:
      prefix: "chore"
      include: "scope"
    groups:
      patch-and-minor:
        update-types:
          - "patch"
          - "minor"
      major:
        update-types:
          - "major"
```

**Comment by greysteil** [comment]
@sklakegg I haven't work on Dependabot for a few years - if you open a new issue the team at GitHub should be able to take a look, though.

**Comment by sklakegg** [comment]
Thanks 👍

**Comment by Waleed2660** [comment]
Hey @sklakegg , I'm currently facing same issue where Dependabot is closing PRs once first one is merged. Did you find any workaround OR did you reach out to the Devs?

**Comment by sklakegg** [comment]
Hey @Waleed2660 
I made this issue: https://github.com/dependabot/dependabot-core/issues/11439
Did not receive any feedback yet, so I went back to just having 1 group and that fixed the bug temporarily

**Comment by Waleed2660** [comment]
Thanks @sklakegg, I encountered this issue when I had a fallback group using "*" pattern. So I removed that group. Now I've got 4 groups & Dependabot worked fine & didn't disregard my grouping policy once any PR is merged. The only downside is that I had to made sure that all dependencies in my project are covered by my grouping policy which is a bit of manual effort but probably worth it
